package.json

```diff
@@ -11,7 +11,7 @@
     "start": "cross-env NODE_ENV=production node app.js",
     "dev": "cross-env NODE_ENV=development node app.js",
     "lint": "eslint .",
-    "test": "mocha"
+    "test": "./node_modules/.bin/istanbul test ./node_modules/.bin/_mocha -- -R spec"
```

Comment:
You don't need the `./node_modules/.bin/` here; NPM scripts add that folder to the `$PATH`, so you can just `istanbul test _mocha -- -R spec`.